# -*- coding: utf-8 -*-
# Generated by Django 1.11.9 on 2018-07-05 23:46
from __future__ import unicode_literals

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('mercado_vindu', '0041_auto_20180613_1110'),
    ]

    operations = [
        migrations.CreateModel(
            name='ProcesoBatchError',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('model_nombre', models.CharField(blank=True, max_length=255, null=True)),
                ('model_id', models.IntegerField(blank=True, null=True)),
                ('linea', models.TextField(blank=True, null=True)),
                ('linea_numero', models.IntegerField(blank=True, null=True)),
                ('error', models.CharField(max_length=255)),
                ('date_created', models.DateTimeField(auto_now_add=True)),
                ('date_updated', models.DateTimeField(auto_now=True)),
            ],
            options={
                'verbose_name': 'ProcesoBatchError',
                'verbose_name_plural': 'ProcesoBatchErrores',
            },
        ),
        migrations.AddField(
            model_name='procesobatch',
            name='procesado_error',
            field=models.BooleanField(default=False),
        ),
        migrations.AddField(
            model_name='procesobatch',
            name='procesado_error_tipo',
            field=models.CharField(blank=True, choices=[('archivo', 'Archivo'), ('registro', 'Registro')], max_length=42, null=True),
        ),
        migrations.AddField(
            model_name='procesobatcherror',
            name='proceso_batch',
            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='procesobatch_error', to='mercado_vindu.ProcesoBatch'),
        ),
    ]
